src/components/SlateEditor/plugins/concept/block/index.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2019-present, NDLA.
```

Comment:
Nye filer kan jo få nytt årstall.
```suggestion
 * Copyright (c) 2022-present, NDLA.
```